slangpy/tests/device/test_cluster_acceleration_structure.py

```
@@ -0,0 +1,400 @@
+    )
+    command_encoder = device.create_command_encoder()
+    command_encoder.execute_cluster_operation(explicit_desc)
+    device.submit_command_buffer(command_encoder.finish())
+    device.wait_for_idle()
+
+    cluster_handle = int(destination_buffer.to_numpy().view(np.uint64)[0])
+    assert cluster_handle == arena.device_address
+
+
+@pytest.mark.parametrize("device_type", helpers.DEFAULT_DEVICE_TYPES)
+def test_cluster_acceleration_structure_trace(device_type: spy.DeviceType) -> None:
```

Ruff (0.16.3)

[warning] Too many statements (60 > 50)

(PLR0915)